src/Cache.cpp

```diff
@@ -46,10 +46,16 @@ static const std::string_view OLM_ACCOUNT_KEY("olm_account");
 static const std::string_view CACHE_FORMAT_VERSION_KEY("cache_format_version");
 static const std::string_view CURRENT_ONLINE_BACKUP_VERSION("current_online_backup_version");
 
+constexpr auto MAX_DBS = 32384UL;
```

Comment:
I just realized I moved that one but not BATCH_SIZE. Do you prefer them above or below?

Reply:
I guess both on one side is nicer, but please leave an empty line between that and the #if